lib/mumukit/flow/suggesting.rb

```diff
@@ -0,0 +1,38 @@
+module Mumukit::Flow
+  module Suggesting
+    def next_suggested_item_for(submitter)
+      set_adaptive_assignment!(submitter)
+      next_item_suggestion(submitter).item
+    end
+
+    private
+
+    def next_item_suggestion(submitter)
+      if end_reached?
+        if pending_siblings?
+          Mumukit::Flow::Suggestion::Revisit.new(first_pending_sibling)
+        else
+          Mumukit::Flow::Suggestion::None
+        end
+      else
+        if should_skip_next_item?
+          Mumukit::Flow::Suggestion::Skip.new(next_item.next_suggested_item_for(submitter))
+        else
+          Mumukit::Flow::Suggestion::Continue.new(next_item)
+        end
+      end
+    end
+
+    def should_skip_next_item?
+      similar_easy_siblings_for_every_tag? unless next_item.learning?
+    end
+
+    def similar_easy_siblings_for_every_tag?
+      next_item.tags.all? { |tag| easy_siblings_with(tag).count >= 2 }
```

Comment:
This `2` is an arbitrary number, right? Perhaps it should be extracted to a method o constant to make it easier to customize

Reply:
I thought that 1 was too little and 3 was too much for the average guide. But it's definitely a number that we can tweak later on. I'll extract it to a method, because we might even choose to try and calculate it on some other property - number of exercises in the guide, or maybe the number of exercises with the same tags, or something else we can come up with.